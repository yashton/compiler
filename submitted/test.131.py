def get
 gjfgj
  vhjghj
   ftgdfgdf
kljkljk