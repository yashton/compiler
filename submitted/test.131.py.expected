(KEYWORD def)
(ID "get")
(NEWLINE)
(INDENT)
(ID "gjfgj")
(NEWLINE)
(INDENT)
(ID "vhjghj")
(NEWLINE)
(INDENT)
(ID "ftgdfgdf")
(NEWLINE)
(DEDENT)
(DEDENT)
(DEDENT)
(ID "kljkljk")
(ENDMARKER)
